src/styles/filter.scss

```diff
@@ -367,9 +367,9 @@
 	gap: 12px;
 	font-size: 14px;
 
-	.filters-history-open {
-		@include improved-clickable(#fff, 10px);
-	}
+	// .filters-history-open {
+	// 	@include improved-clickable(var(--background-primary), 10px);
+	// }
```

Comment:
not sure, whether this line is functional. getting rid of it doesn't change much for filter-history icon in Events.

Reply:
you can delete it, because currently history icon is not in filter-controls div

Reply:
delete it, cause you dont have class 'filter-controls filters-history-open'